dependencies.xml

```diff
@@ -3,7 +3,7 @@
     <!-- fix the version of pyomo, this is because, in the version after 5.7.3, PySP package is moved to a independent
     lib. See https://pyomo.readthedocs.io/en/latest/modeling_extensions/stochastic_programming.html
     We need to update LOGOS to use either PySP or mpi-sppy, using 5.7 which will consistent with HERON-->
-    <pyomo source="forge">5.7.0</pyomo>
+    <pyomo source="forge"/>
```

Comment:
should we force the new pyomo version?

Reply:
@mandd I would recommend that we do not fix the version. Since currently LOGOS works for both 5.7.x and 6.x.x version. In addition, our other plugins have some fixed versions, we can reduce the conflict between the versions for Pyomo. It also tells us when we need to move to use mai-sppy in the future.